cobc/ChangeLog

```diff
@@ -1,3 +1,10 @@
+2022-02-20  Nicolas Berthier <nicolas.berthier@ocamlpro.com>
```

Comment:
2022-02? If that change is _really_ that old please move the entry, but I guess it is just a typo.